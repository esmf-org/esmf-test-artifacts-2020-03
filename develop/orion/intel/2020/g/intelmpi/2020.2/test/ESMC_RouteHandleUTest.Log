build time -- 2022-02-03 02:34:37
20220203 040022.365 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220203 040022.368 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220203 040022.368 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220203 040022.368 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220203 040022.368 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET0 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-10-gce27d44
20220203 040022.368 INFO             PET0 ESMF library build date/time: "Feb  3 2022" "02:32:59"
20220203 040022.369 INFO             PET0 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220203 040022.369 INFO             PET0 ESMF_COMM                   : intelmpi
20220203 040022.371 INFO             PET0 ESMF_MOAB                   : enabled
20220203 040022.371 INFO             PET0 ESMF_LAPACK                 : enabled
20220203 040022.371 INFO             PET0 ESMF_NETCDF                 : enabled
20220203 040022.371 INFO             PET0 ESMF_PNETCDF                : disabled
20220203 040022.372 INFO             PET0 ESMF_PIO                    : enabled
20220203 040022.372 INFO             PET0 ESMF_YAMLCPP                : enabled
20220203 040022.374 INFO             PET0 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220203 040022.374 INFO             PET0 NUMBER_OF_PROCESSORS 4
20220203 040032.648 INFO             PET0 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220203 040032.940 INFO             PET0 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220203 040032.943 INFO             PET0 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220203 040032.943 INFO             PET0 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220203 040032.943 INFO             PET0 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220203 040032.943 INFO             PET0 Finalizing ESMF
20220203 040022.365 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220203 040022.368 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220203 040022.368 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220203 040022.368 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220203 040022.368 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET1 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-10-gce27d44
20220203 040022.368 INFO             PET1 ESMF library build date/time: "Feb  3 2022" "02:32:59"
20220203 040022.369 INFO             PET1 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220203 040022.369 INFO             PET1 ESMF_COMM                   : intelmpi
20220203 040022.371 INFO             PET1 ESMF_MOAB                   : enabled
20220203 040022.371 INFO             PET1 ESMF_LAPACK                 : enabled
20220203 040022.371 INFO             PET1 ESMF_NETCDF                 : enabled
20220203 040022.371 INFO             PET1 ESMF_PNETCDF                : disabled
20220203 040022.372 INFO             PET1 ESMF_PIO                    : enabled
20220203 040022.372 INFO             PET1 ESMF_YAMLCPP                : enabled
20220203 040022.374 INFO             PET1 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220203 040022.374 INFO             PET1 NUMBER_OF_PROCESSORS 4
20220203 040032.649 INFO             PET1 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220203 040032.951 INFO             PET1 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220203 040032.951 INFO             PET1 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220203 040032.951 INFO             PET1 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220203 040032.951 INFO             PET1 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220203 040032.951 INFO             PET1 Finalizing ESMF
20220203 040022.365 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220203 040022.368 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220203 040022.368 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220203 040022.368 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220203 040022.368 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET2 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-10-gce27d44
20220203 040022.368 INFO             PET2 ESMF library build date/time: "Feb  3 2022" "02:32:59"
20220203 040022.369 INFO             PET2 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220203 040022.369 INFO             PET2 ESMF_COMM                   : intelmpi
20220203 040022.371 INFO             PET2 ESMF_MOAB                   : enabled
20220203 040022.371 INFO             PET2 ESMF_LAPACK                 : enabled
20220203 040022.371 INFO             PET2 ESMF_NETCDF                 : enabled
20220203 040022.371 INFO             PET2 ESMF_PNETCDF                : disabled
20220203 040022.372 INFO             PET2 ESMF_PIO                    : enabled
20220203 040022.372 INFO             PET2 ESMF_YAMLCPP                : enabled
20220203 040022.374 INFO             PET2 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220203 040022.375 INFO             PET2 NUMBER_OF_PROCESSORS 4
20220203 040032.661 INFO             PET2 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220203 040032.940 INFO             PET2 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220203 040032.960 INFO             PET2 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220203 040032.960 INFO             PET2 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220203 040032.960 INFO             PET2 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220203 040032.960 INFO             PET2 Finalizing ESMF
20220203 040022.365 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220203 040022.368 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220203 040022.368 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220203 040022.368 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220203 040022.368 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220203 040022.368 INFO             PET3 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-10-gce27d44
20220203 040022.368 INFO             PET3 ESMF library build date/time: "Feb  3 2022" "02:32:59"
20220203 040022.369 INFO             PET3 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220203 040022.369 INFO             PET3 ESMF_COMM                   : intelmpi
20220203 040022.371 INFO             PET3 ESMF_MOAB                   : enabled
20220203 040022.371 INFO             PET3 ESMF_LAPACK                 : enabled
20220203 040022.371 INFO             PET3 ESMF_NETCDF                 : enabled
20220203 040022.371 INFO             PET3 ESMF_PNETCDF                : disabled
20220203 040022.372 INFO             PET3 ESMF_PIO                    : enabled
20220203 040022.372 INFO             PET3 ESMF_YAMLCPP                : enabled
20220203 040022.374 INFO             PET3 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220203 040022.374 INFO             PET3 NUMBER_OF_PROCESSORS 4
20220203 040032.663 INFO             PET3 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220203 040032.964 INFO             PET3 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220203 040032.964 INFO             PET3 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220203 040032.964 INFO             PET3 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220203 040032.965 INFO             PET3 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220203 040032.965 INFO             PET3 Finalizing ESMF
