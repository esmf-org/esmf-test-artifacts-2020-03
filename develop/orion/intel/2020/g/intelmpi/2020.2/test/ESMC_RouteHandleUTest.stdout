build time -- 2022-02-03 02:34:37
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
NUMBER_OF_PROCESSORS 4
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
NUMBER_OF_PROCESSORS 4
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
 PET 2 Test Elapsed Time  -353.788000 msec.
 PET 3 Test Elapsed Time  -353.690000 msec.
 PET 1 Test Elapsed Time  -351.425000 msec.
 PET 0 Test Elapsed Time  -350.732000 msec.
